### PR TITLE
Add the ability to configure a TVL warning

### PR DESCRIPTION
**Pull request by mateuszradomski**
Resolves L2B-4239

### CODE REVIEW

File: packages/frontend/src/components/table/props/getScalingTableColumnsConfig.tsx [hunk 1]
```diff
@@ -88,21 +89,21 @@ export function getActiveScalingSummaryColumnsConfig() {
       noPaddingRight: true,
       headClassName: '-translate-x-[72px]',
       getValue: (project) =>
-        project.slug === 'polygonzkevm' ? (
+        project.tvlWarning !== undefined ? (
```
Comment (torztomasz) [inline packages/frontend/src/components/table/props/getScalingTableColumnsConfig.tsx:92]: lets extract it to`TotalCell` component or smth

File: packages/config/src/layer2s/types/Layer2.ts [hunk 2]
```diff
@@ -88,3 +90,10 @@ export interface Layer2Config extends ScalingProjectConfig {
   /** Configuration for getting finality data */
   finality?: Layer2FinalityConfig
 }
+
+export interface Layer2TVLWarning {
+  /** Content of the warning */
+  content: string
+  /** Color with which the warning should be displayed */
+  severity: "yellow" | "red"
```
Comment (torztomasz) [inline packages/config/src/layer2s/types/Layer2.ts:98]: lets use `sentiment: Pick<Sentiment, 'yellow' | 'red'>`

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/types/Layer2.ts [hunk 3]
```diff
@@ -95,5 +95,5 @@ export interface Layer2TVLWarning {
   /** Content of the warning */
   content: string
   /** Color with which the warning should be displayed */
-  severity: "yellow" | "red"
+  sentiment: 'bad' | 'warning'
```
Comment (torztomasz) [inline packages/config/src/layer2s/types/Layer2.ts]: lets pick it from Sentiment type: `Pick<Sentiment, 'bad' | 'warning'>`

**Review by torztomasz — APPROVED** [review]

File: packages/config/src/layer2s/types/Layer2.ts (hunk 3 above)
Comment (mateuszradomski) [reply packages/config/src/layer2s/types/Layer2.ts]: You can't pick from a union type only from an interface/class

Comment (torztomasz) [reply packages/config/src/layer2s/types/Layer2.ts]: Extract was the correct utility type, sorry 👍

**Pull request merged**